pkg/types/types.go

```diff
@@ -116,6 +117,10 @@ type BasicRuntimeMetadata struct {
 	// ContainerName is the container name. In the case the container runtime
 	// response with multiple containers, ContainerName contains only the first element.
 	ContainerName string `json:"containerName,omitempty" column:"containerName,template:container"`
+
+	// ImageName is the name of the container image where the event comes from
+	// i.e. docker.io/library/busybox:latest
+	ImageName string `json:"imageName,omitempty" column:"imageName" columnTags:"runtime"`
```

Comment:
We can `ContainerName` and `ContainerID`, so I think this should be `ContainerImage`.